tests/acceptance/features/webUIProvisioning/users.feature

```diff
@@ -68,17 +69,85 @@ Feature: add users
+    #to set Quota we can just misuse any LDAP text field
+    Given LDAP config "LDAPTestId" has key "ldapQuotaAttribute" set to "employeeNumber"
+    And the LDAP users have been resynced
+    And the administrator sets the quota of user "user0" to "13 MB" using the webUI
+    Then the quota definition of user "user0" should be "13 MB"
+    And the quota of user "user0" should be set to "13 MB" on the webUI
+    When the LDAP users are resynced
+    And the administrator reloads the users page
+    Then the quota definition of user "user0" should be "13 MB"
+    And the quota of user "user0" should be set to "13 MB" on the webUI
+
+  Scenario: admin sets quota of user for which no LDAP quota attribute is specified but the default quota setting is set
+    #to set Quota we can just misuse any LDAP text field
+    Given LDAP config "LDAPTestId" has key "ldapQuotaAttribute" set to "employeeNumber"
+    And LDAP config "LDAPTestId" has key "ldapQuotaDefault" set to "10MB"
+    And the LDAP users have been resynced
+    When the administrator sets the quota of user "user0" to "13 MB" using the webUI
+    Then the quota definition of user "user0" should be "13 MB"
+    And the quota of user "user0" should be set to "13 MB" on the webUI
+    When the LDAP users are resynced
+    And the administrator reloads the users page
+    Then the quota definition of user "user0" should be "10MB"
+    And the quota of user "user0" should be set to "10MB" on the webUI
+
+  Scenario: admin sets quota of user in LDAP when the default quota setting is set
+    #to set Quota we can just misuse any LDAP text field
+    Given LDAP config "LDAPTestId" has key "ldapQuotaAttribute" set to "employeeNumber"
+    And LDAP config "LDAPTestId" has key "ldapQuotaDefault" set to "10MB"
+    And the LDAP users have been resynced
+    When the administrator sets the ldap attribute "employeeNumber" of the entry "uid=user0,ou=TestUsers" to "13 MB"
+    And the administrator reloads the users page
+    Then the quota of user "user0" should be set to "10MB" on the webUI
+    When the LDAP users are resynced
+    And the administrator reloads the users page
+    Then the quota of user "user0" should be set to "13 MB" on the webUI
+
+  @issue-core-33186
+  Scenario: admin sets quota of user when the default quota setting is set and quota LDAP attribute is specified
+    #to set Quota we can just misuse any LDAP text field
+    Given LDAP config "LDAPTestId" has key "ldapQuotaAttribute" set to "employeeNumber"
+    And LDAP config "LDAPTestId" has key "ldapQuotaDefault" set to "10MB"
+    When the administrator sets the ldap attribute "employeeNumber" of the entry "uid=user0,ou=TestUsers" to "11 MB"
+    And the LDAP users are resynced
+    And the administrator sets the quota of user "user0" to "13 MB" using the webUI
+    Then the quota definition of user "user0" should be "13 MB"
+    And the quota of user "user0" should be set to "13 MB" on the webUI
```

Comment:
I think this needs commented-out examples of what the steps should be when the issue is fixed.